### ISSUE TITLE
estimated pricing

### ISSUE THREAD

**Issue by matthewmueller** [segmentio/stack#34]
This looks so sweet :-)

Not sure if i missed this somewhere, but it'd be great to get a sense of the base cost without running any services.

**Comment by yields** [comment]
agreed! i think it'll be nice if terraform did that out of the box.

**Comment by Brzhk** [comment]
@matthewmueller around $200/month AFAIK with idle instances.
